packages/router-plugin/src/router.state.ts

```diff
@@ -0,0 +1,42 @@
+import { RouterStateSnapshot } from '@angular/router';
+import { Action, Selector, State, StateContext } from '@ngxs/store';
+import { RouterAction, RouterCancel, RouterError, RouterNavigation } from './router.actions';
+
+export type RouterStateModel<T = RouterStateSnapshot> = {
```

Comment:
what is that I see! does typescript allow for default generic values now!?!?!

Reply:
@deebloo yes it has been in typescript since 2.3 :)

Reply:
Well son of a b